covid_symptoms/Dockerfile

```diff
@@ -1,34 +1,31 @@
-FROM openjdk:8-alpine
+FROM eclipse-temurin:8-jdk
 
 # Set to your proxies if necessary to build behind firewalls:
 #ENV HTTP_PROXY="http://10.41.13.6:3128"
 #ENV HTTPS_PROXY="http://10.41.13.6:3128"
 #ENV FTP_PROXY="http://10.41.13.6:3128"
 
-RUN apk update && apk add ca-certificates openssl wget unzip subversion maven
-RUN apk upgrade
+RUN apt-get update && apt-get install -y ca-certificates openssl wget unzip subversion maven
 
-RUN apk add --no-cache nss
-## Download apache-tomcat and extract:
-RUN wget -q https://archive.apache.org/dist/tomcat/tomcat-9/v9.0.21/bin/apache-tomcat-9.0.21.zip
-RUN unzip -q apache-tomcat-9.0.21.zip
+RUN apt-get install -y libnss3
 
 ## Check out version of ctakes with best working web-rest module
 ## Then compile with maven
 RUN svn export https://svn.apache.org/repos/asf/ctakes/trunk@1894987 ctakes
 
-WORKDIR /
-COPY covid_symptoms_ctakes.bsv /ctakes/resources/org/apache/ctakes/dictionary/lookup/fast/covid.bsv
-COPY covid_symptoms.xml /ctakes/ctakes-web-rest/src/main/resources/org/apache/ctakes/dictionary/lookup/fast/
-COPY pom.xml /ctakes
-
+WORKDIR /tmp
 ## Copy hsql dictionary descriptor into right location
 RUN wget -q -O dict.zip  https://sourceforge.net/projects/ctakesresources/files/snorx_2021aa.zip/download
-RUN mkdir -p /ctakes/resources/org/apache/ctakes/dictionary/lookup/fast/
+RUN mkdir -p /ctakes/ctakes-web-rest/src/main/resources/org/apache/ctakes/dictionary/lookup/fast/
```

Comment:
Do we need this line? The unzip next line was working before this, yeah?

Reply:
yeah, this is now superfluous.